requirements.txt

```diff
@@ -0,0 +1,10 @@
+numpy
+scipy
+# for documentation
+sphinx==2.1.2
+breathe==4.13.1
+sphinx_rtd_theme
+nbsphinx
+# Developers tools
+jupyter
+cpplint
```

Comment:
I added some libraries, not essential for the tests but for running the tutorials and developer tools

Reply:
I wouldn't call running the tutorials a "developer tool"